<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapBuilderWindow</class>
 <widget class="QMainWindow" name="MapBuilderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>1491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Blocks</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="groundBrickButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/textures/ground_brick.png</normaloff>:/images/textures/ground_brick.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="solidBrickButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/textures/solid_brick.png</normaloff>:/images/textures/solid_brick.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="platformBrickButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/textures/platform_brick.png</normaloff>:/images/textures/platform_brick.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bonusBoxButton">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/textures/bonus_box.png</normaloff>:/images/textures/bonus_box.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>10</y>
      <width>211</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="placeGroundButton">
       <property name="text">
        <string>   Place ground</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/images/textures/ground_brick.png</normaloff>:/images/textures/ground_brick.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1516</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau_niveau"/>
    <addaction name="actionCharger_un_niveau"/>
    <addaction name="actionSauvegarder_le_niveau"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouveau_niveau">
   <property name="text">
    <string>Nouveau niveau</string>
   </property>
  </action>
  <action name="actionCharger_un_niveau">
   <property name="text">
    <string>Charger un niveau</string>
   </property>
  </action>
  <action name="actionSauvegarder_le_niveau">
   <property name="text">
    <string>Sauvegarder le niveau</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
